core/domain/exp_services_test.py

```diff
@@ -2180,6 +2184,28 @@ def setUp(self):
 
         rights_manager.publish_exploration(self.ALBERT_ID, self.EXP_ID_2)
 
+        exploration_3 = self.save_new_valid_exploration(
+            self.EXP_ID_3, self.ALBERT_ID)
+
+        rights_manager.publish_exploration(self.ALBERT_ID, self.EXP_ID_3)
+
+        exp_services.delete_exploration(self.ALBERT_ID, self.EXP_ID_3)
+
+    def test_get_displayable_exploration_summaries_matching_ids(self):
+        # A list of exp_id's are passed in:
+        # EXP_ID_1 -- private exploration
+        # EXP_ID_2 -- pubished exploration
+        # EXP_ID_3 -- deleted exploration
+        # Should only return [EXP_ID_2]
+
+        displayable_summaries = (
+            exp_services.get_displayable_exploration_summaries_matching_ids([
+                self.EXP_ID_1, self.EXP_ID_2, self.EXP_ID_3], self.ALBERT_ID))
+
+        self.assertEqual(
```

Comment:
Please check other parts of these dicts. At least check that the keys match what you expect, and that all values (except for last_updated_msec, which it's hard to get the expected value of) are correct.